### DISCUSSION TITLE
Can SchaleDB add a tool for arranging TL?

### DISCUSSION THREAD

**Discussion by Thinkersn** [lonqie/SchaleDB#101] [Ideas and Suggestions]
The current damage calculator has been very helpful for the arrangement of timeline in raids and GA, but since SchaleDB has already collected all the skill data, is it possible to upgrade the existing damage calculator to a timeline calculator? Specifically, for each BOSS, this calculator would allow for the free combination of a team and the free dragging of students' or BOSS's buff skills to a certain position on the timeline. It would also be able to freely calculate the EX damage during a specific period on the timeline and the total damage of the entire team up to a certain point in time. This would greatly assist in the arrangement of timeline in raids by making it more intuitive to discover when a certain buff runs out and what the total damage of a timeline is at a specific time.

**Comment by lonqie** [comment]
Thanks for the suggestion. I think what you are describing is very complex and goes far beyond the scope of what the existing calculator was originally designed to do. It would require a significant amount of work to make this possible even on a basic level.

Introduce time and you're effectively simulating most of the game at that point. You've got a lot more to consider regarding skill timings, auto attacks (DPS already uses a simplified calculation that doesn't factor skill activations). There's so much room for inaccuracies and the amount of mechanics you'd have to simplify or leave up to the user to define would make me question if the results one would get out of it would even be useful.

**Reply by Thinkersn** [reply]
I see. Thanks for your reply!
